README.md

```diff
@@ -378,6 +378,12 @@ Here's an example of mixing them all together:
 ospec '**/*.test.js' --ignore 'folder1/**' --require esm ./my-file.js
 ```
 
+#### For NodeJS v13+ only
```

Comment:
The exact version may be lower... I've tried it with some version of node v12 (can't remember which) without success though, before updating to the latest.